conda/recipes/cudf/meta.yaml

```diff
@@ -64,6 +64,7 @@ requirements:
     - rapids-build-backend >=0.3.0,<0.4.0.dev0
     - scikit-build-core >=0.7.0
     - dlpack >=0.8,<1.0
+    # TODO: Change to `2.0` for NumPy 2
     - numpy 1.23
```

Comment:
Once we update to NumPy 2, that will be true

Right now we build with NumPy 1.23, which means [`run_exports` adds `{{ pin_subpackage("numpy") }}`]( https://github.com/conda-forge/numpy-feedstock/blob/ddbe092d07a0a6f57c3fd6854d7c17062c8fa784/recipe/meta.yaml#L26-L27 ). The [defaults of `pin_subpackage`]( https://docs.conda.io/projects/conda-build/en/stable/resources/variants.html#extra-jinja2-functions ) are to tightly pin on the lower bound and pin the upper bound based on major version. So this turns into `numpy >=1.23.5,<2.0a0`

Reply:
oh got it, right. Ok sounds good to me.